client/locales/en-US.json

```diff
@@ -0,0 +1,226 @@
+         "unban":"Unban ({{unbanMode}}) from the current channel. This can be a nickname or a hostmask.",
+         "voice":"Give voice ({{voiceMode}}) to one or several users in the current channel.",
+         "whois":"Retrieve information about the given user on the current network."
+      },
+      "about":{
+         "title":"About The Lounge",
+         "running_from_source":"The Lounge is running from source",
+         "version":"The Lounge is in version ",
+         "release_notes":"See release notes",
+         "website":"Website",
+         "documentation":"Documentation",
+         "report_bug":"Report a bug"
+      }
+   },
+   "client":{
+      "close_channel":"Close",
+      "toggle_chan_list":"Toggle channel list",
+      "toggle_user_list":"Toggle user list",
+      "open_context_menu":"Open the context menu",
+      "show_older_messages":"Show older messages",
+      "search_among_user_list":"Search among the user list",
+      "disconnect_from":"Disconnect from {{server}}?",
+      "toggle_prefetched_media":"Toggle prefetched media",
+      "ban_list":{
+         "banned":"Banned",
+         "banned_by":"Banned By",
+         "banned_at":"Banned At"
+      },
+      "channel_list":{
+         "channel":"Channel",
+         "users":"Users",
+         "topic":"Topic"
+      },
+      "invited":{
+         "invited":"invited",
+         "you":"you",
+         "to":"to"
+      },
+      "open_image": "Open image",
+      "previous_image": "Previous image",
+      "next_image": "Next image",
+      "visit_page": "Visit page",
+      "toggle_image_preview": "Toggle image preview",
+      "toggle_website_preview": "Toggle website preview",
+      "has_joined":"has joined the channel",
+      "has_kicked":"has kicked",
+      "sets_mode":"sets mode",
+      "now_known_as":"is now known as",
+      "has_left":"has left the channel",
+      "has_quit":"has quit",
+      "topic":{
+         "has_changed":"has changed the topic to",
+         "topic_is":"The topic is:"
+      },
+      "topic_set_by":{
+         "set_by":"Topic set by",
+         "on":"on"
+      },
+      "no_such_nick":"No such nick: {{nick}}",
+      "connection_failed":"Connection failed: {{data}}",
+      "connecting":"Connecting...",
+      "finalizing_connection":"Finalizing connection...",
+      "authorized":"Authorized, loading messages...",
+      "exit_warning":"Are you sure you want to navigate away from this page?"
+   },
+   "server":{
+      "page_closed":"Page closed",
+      "password":{
+         "change_enter_new":"Please enter a new password",
+         "change_fail":"Failed to update your password",
+         "change_fail_entered_not_match":"Both new password fields must match",
+         "change_fail_entered_not_match_account":"The current password field does not match your account password",
+         "change_success":"Successfully updated your password, all your other sessions were logged out"
```

Comment:
I believe since #1199, sessions are no longer logged out, so this should probably be updated

Reply:
I don't see any strings being changed in that PR, so I think this is fine

Reply:
fair enough